### PR TITLE
Add git commit timestamp to jib creationTimestamp

### PR DESCRIPTION
**Pull request by vedantchokshi**

### CODE REVIEW

**Review by massdosage — CHANGES_REQUESTED** [review]
Looks good, we just need a section in the README explaining how to use this downstream.

File: README.md [hunk 1]
```diff
@@ -10,6 +10,23 @@ You can obtain this POM from Maven Central:
 The Expedia Group OSS Parent POM provides a Maven parent POM that is intended to be used by Expedia Group projects that require certain 
 base functionality and need to publish artifacts to [Maven Central](https://search.maven.org/).
 
+# Plugins
+## [Jib Maven plugin](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin)
+This plugin builds an optimized Docker image for a Java application and uploads it to Artifactory.
```
Comment (massdosage) [inline README.md]:
```suggestion
This plugin builds an optimized Docker image for a Java application and uploads it to [Docker Hub](https://hub.docker.com/).
```

File: README.md [hunk 2]
```diff
@@ -10,6 +10,23 @@ You can obtain this POM from Maven Central:
...
+```xml
+<plugin>
+    <groupId>com.google.cloud.tools</groupId>
+    <artifactId>jib-maven-plugin</artifactId>
+</plugin>
+<plugin>
+    <groupId>pl.project13.maven</groupId>
+    <artifactId>git-commit-id-plugin</artifactId>
+</plugin>
+```
```
Comment (massdosage) [inline README.md:28]: We should also have a table here showing the most common properties that one might want to override like we do internally.

File: README.md [hunk 3]
```diff
@@ -10,6 +10,23 @@ You can obtain this POM from Maven Central:
...
+
```
Comment (massdosage) [inline README.md:29]: We should also have the `Build Docker Image` section, minus the first bit in our internal version which discusses non-OSS builds, the rest is still valid for disabling the plugin and setting the timeout.

**Review by massdosage — CHANGES_REQUESTED** [review]

**Review by massdosage — APPROVED** [review]
LGTM

**Review by javsanbel2 — APPROVED** [review]

**Pull request merged**